### PR TITLE
Update templates to use the integrated FSharp compiler

### PR DESCRIPTION
**Pull request by KevinRansom**
Hi this PR, removes the FSharp.Sdk from the headers of all the F# templates.  It also removes the package references to fsharp.compiler.tools and the FSharp.SDK.

### CODE REVIEW

File: template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/StarterWeb-FSharp/Company.WebApplication1.fsproj [hunk 1]
```diff
@@ -15,9 +15,9 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.All" Version="2.0.0-preview1-24801" />
     <PackageReference Include="FSharp.Core" Version="4.1.*" />
     <PackageReference Include="FSharp.NET.Sdk" Version="1.0.*" PrivateAssets="All" />
```
Comment (mlorbetske) [inline template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/StarterWeb-FSharp/Company.WebApplication1.fsproj]: These two should be removed, right?

Comment (KevinRansom) [inline template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/StarterWeb-FSharp/Company.WebApplication1.fsproj]: yes, thanks

File: template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/StarterWeb-FSharp/Company.WebApplication1.fsproj [hunk 2]
```diff
@@ -15,9 +15,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="FSharp.Core" Version="4.1.*" />
-    <PackageReference Include="FSharp.NET.Sdk" Version="1.0.*" PrivateAssets="All" />
-    <PackageReference Include="Microsoft.AspNetCore.All" Version="2.0.0-preview1-final" />
+    <PackageReference Include="Microsoft.AspNetCore.All" Version="2.0.0-preview1-24801" />
```
Comment (mlorbetske) [inline template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/StarterWeb-FSharp/Company.WebApplication1.fsproj]: This should refer to 2.0.0-preview1-final. Beyond that, let's rebase & then we can merge (assuming that the `cla-required` status gets resolved).

**Comment by KevinRansom** [comment]
@mlorbetske well since I am an employee, the CLA is not required.  I think there is a some issue with the checker.

**Pull request merged**